Prac_09/cleanup_files.py

```diff
@@ -0,0 +1,44 @@
+"""
+Cleanup inconsistent song lyrics file names
+"""
+import os
+
+
+def main():
+    """Cleanup inconsistent song lyrics file names."""
+    print("Starting directory is: {}".format(os.getcwd()))
+    # Change to desired directory
+    os.chdir('Lyrics/Christmas')
+    for directory_name, subdirectories, filenames in os.walk('.'):
+        print("Directory:", directory_name)
+        for filename in filenames:
+            new_name = get_fixed_filename(filename)
+            print("Renaming {} to {}".format(filename, new_name))
+            full_name = os.path.join(directory_name, filename)
+            new_name = os.path.join(directory_name, new_name)
+            os.rename(full_name, new_name)
+
+
+def get_fixed_filename(filename):
+    """Return a 'fixed' version of filename."""
+    filename = filename.replace(" ", "_").replace(".TXT", ".txt")
+    new_name = ""
+    for i, character in enumerate(filename):
+        if i != 0:
```

Comment:
You should be able to find a more efficient way of handling this. This if statement is the same every time except the first iteration.